### PR TITLE
Rebranding of the project

### PR DESCRIPTION
**Pull request by dex4er**
* Icons changed
* URLs changed
* Removed all mkdocs as we do not host it nor use the pipelines to build it
* Removed configuration for ADRs
* Removed docs with outdated links
* Removed code of conduct as we need a total new one
* Renamed open-lens to freelens directory
* Renamed @k8slens to @freelensapp nodejs packages namespace
* Renamed engine lens in `package.json` of plugins to freelens
* Renamed directory for plugins to `~/freelens`
* All versions renumbered to `0.1.0`.

### CODE REVIEW

**Comment by dex4er** [comment]
This huge PR completes #3 except email.

**Comment by dex4er** [comment]
Email is rebranded too

**Comment by dex4er** [comment]
I'm going to renumber packages from 6.5.0 and others back to 0.1.0

**Comment by dex4er** [comment]
In the result: changing NPM namespace and version breaks the compatibility for Lens plugins. However, I don't see any sense to have it as it was compatible only for major versions.

**Review by robertobandini — CHANGES_REQUESTED** [review]

File: packages/business-features/keyboard-shortcuts/package.json
[the remark this answers is not in the document]
Comment (dex4er) [reply packages/business-features/keyboard-shortcuts/package.json:1]: It looks like. Ok I'll check if there are some leftovers and we're ready for merge

[the remark this answers is not in the document]
Comment (dex4er) [reply packages/business-features/keyboard-shortcuts/package.json:1]: Updated all package.json files

**Review by robertobandini — APPROVED** [review]

**Pull request merged**